docs/workflows/genomic_characterization/theiaeuk.md

```diff
@@ -92,6 +92,10 @@ All input reads are processed through "core tasks" in the TheiaEuk workflows. Th
     These tasks are performed regardless of the organism and provide quality control and taxonomic assignment.
 
 {{ include_md("common_text/gambit_task.md") }}
+
+!!! warning ""
```

Comment:
add this to the actual task MD so it shows up whenever we run busco -- you can put it in a conditional

Reply:
that way we don't forget where it is in the future